pytest_opentmi/OpenTmiReport.py

```diff
@@ -130,6 +130,14 @@ def _update_teardown_result(self, report):
         # update result with updated last_result
         self.results[-1] = last_result
 
+    def _update_user_properties(self, report):
+        if report.user_properties:
+            last_result = self.results[-1]
+            if not 'properties' in last_result.execution.profiling:
+                last_result.execution.profiling['properties'] = {}
+            for (key, value) in report.user_properties:
+                last_result.execution.profiling['properties'][key] = value
```

Comment:
does this depends on any other pytest plugin now? If I remember correctly it originally was in client because it require other plugin to be available, but might remember incorrectly.. 🤔

Reply:
The same is done in _new_result https://github.com/OpenTMI/pytest-opentmi/blob/2d1f2822235716f8aab935fd271597f70d6bb439/pytest_opentmi/OpenTmiReport.py#L173. This just adds the functionality to teardown step as well.

Reply:
could that utilise same function?

Reply:
Sure.